resources/lang/en/contest.php

```diff
@@ -30,6 +30,7 @@
         '_' => 'Judging results',
         'creator' => 'creator',
         'score' => 'Score',
+        'score_std' => 'Standardised Score',
         'total_score' => 'total score',
```

Comment:
casing?

Reply:
this one of part of `Score` not `total score`